letsencrypt_transip_nl/README.rst

```diff
@@ -0,0 +1,80 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============================
+LetsEncrypt transip.nl support
+==============================
+
+This module was written to extend the functionality of letsencrypt module
+to support transip.nl DNS challenges. 
+
+Upon receiving a challenge of type DNS, this module will automatically create
+the relevant DNS record on your domain.
+
+Installation
+============
+
+To install this module, you need to:
+
+Follow the usuall installation procedure for a module in your Odoo version.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+Go to Settings -> General Settings and fill the required fields under
+LetsEncrypt.
```

Comment:
we do expect people to know what they're doing, but here i think you can add a few words on what to fill in and where to acquire the values